src/Controllers/ElementalAreaController.php

```diff
@@ -139,7 +139,7 @@ public function save(array $data, Form $form): HTTPResponse
         $dataWithoutNamespaces = static::removeNamespacesFromFields($data, $element->ID);
 
         // Update and write the data object which will trigger model validation
-        $element->update($dataWithoutNamespaces);
+        $element->updateFromFormData($dataWithoutNamespaces);
```

Comment:
Using updateFromFormData() goes on a slightly longer code path that includes FormField::setSubmittedValue() which means that field values submitted as arrays get handled correctly, which is the case for UploadField and SearchableDropdownField,